front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/ModalSuggeredVias.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useContext } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { useTranslation } from 'react-i18next';
-import { FaLongArrowAltUp, FaLongArrowAltDown, FaTrash, FaMinus } from 'react-icons/fa';
+import { FaTrash, FaMinus } from 'react-icons/fa';
```

Comment:
It will be no cost to use octicons already ! Use `react-icons/go` from https://react-icons.github.io/react-icons/icons/go/